vendor/init.bat

```diff
@@ -8,7 +8,7 @@
 
 :: Find root dir
 if not defined CMDER_ROOT (
-    for /f "delims=" %%i in ("%ConEmuDir%\..\..") do set CMDER_ROOT=%%~fi
+    for /f "delims=" %%i in ("%ConEmuDir%\..\..") do set "CMDER_ROOT=%%~fi"
```

Comment:
Are you sure you want to wrap the whole assignment within double quotes? The left side of assignment is basically declaring the variable/identifier, and should not need quotes. The right side instead is what might be affected by double quotes. Just to be sure, have you checked if it works jut with `..... do set CMDER_ROOT="%%~fi"`